hiob-ludolf-centre-for-ethiopian-studies.csl

```diff
@@ -20,9 +20,11 @@
     </contributor>
     <category citation-format="author-date"/>
     <category field="humanities"/>
-    <updated>2017-10-02T08:36:15+00:00</updated>
+    <updated>2021-09-22T12:29:15+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
+ <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
```

Comment:
You're duplicating rights and `</info>` here, that's likely breaking everything else (you've also disabled allowing us to make changes to your pull request, which makes fixing things harder on our end)

Reply:
I did not do this intentionally (disabling the changes) sorry about that.